src/view/components/selectorLearnMoreLink.styl

```diff
@@ -0,0 +1,3 @@
+.SelectorLearnMoreLink {
+  margin-right: 56px;
```

Comment:
Will you add a comment here describing why there's 56px margin-right? Otherwise some future engineer will likely remove/change it.

Reply:
I actually added the comment to the jsx file for both new components